README.md

```diff
@@ -85,7 +88,7 @@ A more convenient way to run tests in development is to use a file watcher to au
 
 ```
 # Run unit test file watcher
```

Comment:
Comment no longer true (as well as adding no value)

Reply:
I agree the unit part isnt true, but the comment is part of a block that shows samples for three different package.json scripts that can be called so would rather leave the comment in, but remove the 'unit' part.

Reply:
My bad, mixing my bash and markdown foo